.circleci/config.yml

```diff
@@ -7,6 +7,11 @@ jobs:
       xcode: "10.0.0"
     steps:
       - checkout
+      - run:
+          name: "Installing Fish"
+          command: |
+            brew update
+            brew install fish
```

Comment:
can you fold the Fish installation into "Preparing Conda Environment" and rename it to "Preparing Environment"?